kdk

```diff
@@ -235,7 +235,6 @@ def destroy(ctx, interactive=True):
     except:
         click.echo("ERROR: Failed to destroy KDK. Confirm the container is running.")
         exit(1)
-    kdk_container.kill()
```

Comment:
I think we need to keep this for when the KDK is running.
maybe try adding a `try/except: pass` to handle both conditions.